plone/app/z3cform/wysiwyg/widget.py

```diff
@@ -36,4 +40,6 @@ def update(self):
 @zope.interface.implementer(z3c.form.interfaces.IFieldWidget)
 def WysiwygFieldWidget(field, request):
     """IFieldWidget factory for WysiwygWidget."""
+    logger.warn("Deprecation Warning\nplone.app.z3cform.wysiwyg.WysiwygFieldWidget "
+                "is deprecated and will be removed in Plone 4.1")
```

Comment:
Oops, should be 5.1
On Aug 29, 2015 1:30 AM, "Gil Forcada Codinachs" <notifications@github.com>
wrote:

> In plone/app/z3cform/wysiwyg/widget.py
> <https://github.com/plone/plone.app.z3cform/pull/21#discussion_r38259670>:
>
> > @@ -36,4 +40,6 @@ def update(self):
> >  @zope.interface.implementer(z3c.form.interfaces.IFieldWidget)
> >  def WysiwygFieldWidget(field, request):
> >      """IFieldWidget factory for WysiwygWidget."""
> > +    logger.warn("Deprecation Warning\nplone.app.z3cform.wysiwyg.WysiwygFieldWidget "
> > +                "is deprecated and will be removed in Plone 4.1")
>
> 4.1 or 5.1?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/plone/plone.app.z3cform/pull/21/files#r38259670>.
>